sites/upsun/src/define-routes/cache.md

```diff
@@ -153,48 +153,45 @@ A full list of HTTP headers is available on [Wikipedia](https://en.wikipedia.org
 
 ### `cookies`
 
-A list of allowed cookie names to include values for in the cache key.
+The `cookies` key allows you to define a list of cookies you want to include in the cache key, if any.
 
-All cookies bypass the cache when using the default (`['*']`) or if the `Set-Cookie` header is present.
-This is done by sending the following header: `X-Platform-Cache: BYPASS`.
+| Possible values          | Description                                                                           | Default |
+|--------------------------|---------------------------------------------------------------------------------------|---------|
+| `['*']`                  | Any request with a cookie bypasses the cache.</br></br> Note that this is achieved by {{% vendor/name %}} adding the `X-Platform-Cache: BYPASS` HTTP header in the router, and that the same behaviour applies if the `Set-Cookie` header is present. | Yes |
+| `[]`                     | Ignore all cookies.                                                                   | No |
+|`['cookie_1','cookie_2']` | A list of allowed cookies to include in the cache key. All other cookies are ignored. | No |
 
-For example, for the cache key to depend on the value of the `foo` cookie in the request.
+#### Example with a single value
+
+With the following configuration, the cache key depends on the value of the `foo` cookie in the request.
 Other cookies are ignored.
 
 ```yaml {configFile="routes"}
-routes:
-    https://{default}/:
-        # ...
-            cache:
-                enabled: true
-                cookies: ["foo"]
+https://{default}/:
```

Comment:
This all looks good to me, but we should keep the top-level `routes` key on the Upsun side, no?

Reply:
Oh absolutely, bad copy-paste on my end